Error during the build process

<!--

If this is a bug, please use the template below. If this is a question or general discussion topic, please start a conversation in our chat https://chat.pop-os.org/ or post on our subreddit https://reddit.com/r/pop_os - as those are the proper forums for that type of discussion.

-->
**Describe the problem:**
I am trying to install pop on a non popOS system, and I was running the build instructions. When I ran the `ninja` command I had few errors show up:
```
bhargav@bhargav-kk:.../gtk-theme/build$ ninja
[50/59] Generating light-gtk-3.20-gtk-dark-generated.css with a custom command.
FAILED: gtk/src/light/gtk-3.0/gtk-dark-generated.css
/usr/bin/sassc -a ../gtk/src/light/gtk-3.0/gtk-dark.scss gtk/src/light/gtk-3.0/gtk-dark-generated.css
Error: It's not clear which file to import for '@import "hdy/pop-dark"'.
       Candidates:
         hdy/_pop-dark.scss
         hdy/pop-dark.scss
         hdy/pop-dark.css
       Please delete or rename all but one of these files.
        on line 12:1 of ../gtk/src/light/gtk-3.0/gtk-dark.scss
>> @import 'hdy/pop-dark';

   ^
[51/59] Generating light-gtk-3.20-gtk-generated.css with a custom command.
FAILED: gtk/src/light/gtk-3.0/gtk-generated.css
/usr/bin/sassc -a ../gtk/src/light/gtk-3.0/gtk.scss gtk/src/light/gtk-3.0/gtk-generated.css
Error: It's not clear which file to import for '@import "hdy/pop-dark"'.
       Candidates:
         hdy/_pop-dark.scss
         hdy/pop-dark.scss
         hdy/pop-dark.css
       Please delete or rename all but one of these files.
        on line 18:1 of ../gtk/src/light/gtk-3.0/gtk.scss
>> @import 'hdy/pop-dark';

   ^
[56/59] Generating gtk_gresource with a custom command.
ninja: build stopped: subcommand failed.
```
<!-- 
Please take a screenshot demonstrating the issue and include it below.
-->

**Steps to reproduce:**
1. Following the steps given in ["Installation from Git Source:"](https://github.com/pop-os/gtk-theme#installation-from-git-source).

**Distribution:**
<!--
run:
    cat /etc/os-release | grep -i pretty_name
-->
Ubuntu 20.04 - WSL2

--- a/gtk/src/light/gtk-3.0/hdy/Pop-dark.scss
+++ b/gtk/src/light/gtk-3.0/hdy/Pop-dark.scss
@@ -1,4 +1,0 @@
-$variant: 'dark';
-$high_contrast: false;
-
-@import 'Pop-base';

